### ISSUE TITLE
Personalized Job Board MVP

### ISSUE THREAD

**Issue by devstein** [shared-recruiting-co/shared-recruiting-co#69]
**Is your feature request related to a problem? Please describe.**
As a user (candidate), I want to see information about my job opportunities within the SRC app

**Describe the solution you'd like**
On `/account/jobs` display, a table in the following format

| Company | Role | Recruiter Name | Recruiter Title | Recruiter Email | Emailed At |
|---------|------|--------------|---------------|---------------|------------|
|         |      |              |               |               |            |
|         |      |              |               |               |            |
|         |      |              |               |               |            |

**Describe alternatives you've considered**
Alternatives are
1) Keep emails in Gmail
2) Force recruiters to fill out job information

For (1), gmails interface isn't easily scannable and hard to compare across different roles. 
For (2), this is a long term goal, but it requires onboarding recruiters, which is out of scope for now.

**Additional context**

- The job information should be parse from email information
- Identify a job by it's thread (a.k.a original) id, but jobs should have an independent uuid
- If an email doesn't have enough of these properties, do not add it to a user job board
- Consider adding a `/account/jobs/[id]` route for viewing the original email
- For now, jobs will be duplicated across users. In the future, we want to de-dupe jobs
- For now, jobs are read-only

#### Schema Ideation

```sql
create table public.user_email_job (
    job_id uuid not null,
    user_id uuid references auth.users(id) not null,
    user_email text not null,
    -- unique email thread ID
    thread_id text not null,
    -- use jsonb while we figure out the job schema
    data jsonb not null default '{}'::jsonb,
    created_at timestamp with time zone not null default now(),
    updated_at timestamp with time zone not null default now(),

    primary key (job_id)
);

```

**Comment by devstein** [comment]
Depends on #50

**Comment by devstein** [comment]
Related article for displaying emails in the browser: https://making.close.com/posts/rendering-untrusted-html-email-safely

**Comment by devstein** [comment]
Moving this to "Done" for now. There will be many follow up tickets to come.
